queries/latex/highlights.scm

```diff
@@ -0,0 +1,161 @@
+;; Math
+[
+ (displayed_equation)
+ (inline_formula)
+] @text.math
+
+;; This highlights the whole environment like vimtex does
+((environment
+  (begin
+   name: (word) @_env)) @text.math
+   (#match? @_env "^(displaymath|eqn|eqnarray|align)[*]?$"))
```

Comment:
`eqn` is not standard, I believe -- it should be `equation`.

Also, `multline` is a (AMSmath) math environment.

Reply:
Also, if these are `TSMath`, wouldn't that be (more) appropriate for `$` and `\[` etc. as well?